packages/ui/src/Slider.module.scss

```diff
@@ -0,0 +1,261 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+// thumb width without a white border, let's make it the height of the slider
+$thumbWidth: c.$slideTrackHeight;
+
+// We've double checked that everything works in IE11
+/* doiuse-disable calc */
+/* doiuse-disable multicolumn */
+/* doiuse-disable css-appearance */
+
+@mixin track() {
+  background: none; // get rid of track background
+  appearance: none;
+  // it's important for these values to be 0 in order to be consistent across (Firefox/IE11) & Chrome
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+@mixin thumb() {
+  box-shadow: 0 c.$sliderThumbShadowSize c.$sliderThumbShadowBlur c.$colorShadow;
+  background: c.$colorPrimaryLight; // background will represent the dot inside
+  border: c.$sliderThumbBorderSize solid c.$colorNeutralWhite; // the white part will be done by the border
+  box-sizing: content-box; /* and because of all of this, we need content-box */
+  border-radius: 50%;
+  pointer-events: auto; /* catch clicks */
+  width: $thumbWidth;
+  height: $thumbWidth;
+}
+
+@mixin gridParent($cols: 1fr, $rows: 1fr) {
```

Comment:
I would suggest not to use grids because of IE11. Yes, they are supported there, but it an older spec, there are plenty of bugs. I just feel like it could seriously backfire. Isn't there a way to do the same styling without grids?
@danad02 @jankoritak @jvorcak @jozefhruska  WDYT?

Reply:
We can do absolute positioning, that should be quite easy.